requirements.txt

```diff
@@ -4,7 +4,8 @@ intake
 xarray
 zarr
 fsspec>=0.7.4
-pydap
+-e git+https://github.com/pydap/pydap#egg=pydap
+-e git+https://github.com/Pylons/webob#egg=webob
```

Comment:
I know the reason for this. But we've had discussions before about the use of not-yet-released libraries. And the general feeling was that we shouldn't rely on those, so this change is at least critical. @RobertPincus maybe has an opinion on this as well.

Reply:
I agree, it's probably good to have this not merge now since the webob-issue will hopefully fixed soon. However, the last official pydap package has been published in 2017 on pypi. Many issues have been solved by now e.g. the handling of gzipped transfers. We could make a fork and publish that on pypi to get a pip package. This is probably the best solution?!

Reply:
Couldn't we persuade the pydap people to update the package? Open a PR on their end?

Reply:
I filed an issue at webob, if / when they are planning to do a release.

Reply:
There will be a new release of `webob` in the upcoming days.

Reply:
Now that there's `netcdf4==1.5.6` and `intake-xarray` supports opendap via netcdf, that might be a feasible way out. See comments below.